src/ExposureHistory/History/index.tsx

```diff
@@ -44,7 +52,18 @@ const History: FunctionComponent<HistoryProps> = ({
 
   const handleOnPressCheckForExposures = async () => {
     setCheckingForExposures(true)
-    await checkForNewExposures()
+    const checkResult = await checkForNewExposures()
+    if (checkResult.kind === "success") {
+      showMessage({
+        message: t("exposure_history.check_successful"),
```

Comment:
what do you think about using the common translations here?  i.e. `common.success`

This is less to expect users to read and fewer translations for us to need to maintain